kubernetes/samples/scripts/create-soa-domain/README.md

```diff
@@ -26,3 +26,10 @@ Refer to the [User guide](https://oracle.github.io/weblogic-kubernetes-operator/
 ### Getting started
 
 The fastest way to experience the operator is to follow the [User guide](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-fmw-domains/soa-suite/).
```

Comment:
The fastest way to experience the operator is to follow the [User guide] -> That's not true. 
The fastest way to experience the operator is to follow the [Quick Start] guide. If you want users to follow the User Guide instead of the Quick Start guide, I suggest that you provide a different sentence, such as, "For detailed information about all aspects of using the operator, see the User Guide."